dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/FeatureRepository.java

```diff
@@ -417,6 +418,108 @@ private void readCache(boolean firstInit) {
         }
     }
 
+    /**
+     * Answer the versioned feature shortName if exists for the passed platform version or null if doesn't exist.
+     *
+     * @return String versionedFeatureName
+     */
+    public String getVersionlessFeatureVersionForPlatform(String versionlessFeatureName, String platformName) {
+        SubsystemFeatureDefinitionImpl versionlessFeature;
+        Map features = getAllFeatures();
+        versionlessFeature = (SubsystemFeatureDefinitionImpl) features.get("io.openliberty.versionless." + versionlessFeatureName);
```

Comment:
May be able to just use `getFeature(versionlessFeatureName)` it checks both the symbolic name and the short name if its a public feature.

Reply:
Thanks! - Yes! - In fact I use that api later on....  haha - change made....